CONTRIBUTING.md

```diff
@@ -22,24 +22,46 @@ See the following directory trees for further recommended examples:
 - [test/built-ins/WeakMap](./test/built-ins/WeakMap/)
 - [test/language/arguments-object](./test/language/arguments-object/)
 
-**Note** The project is currently transitioning from a naming system based on specification section numbers. There remains a substantial number of tests that conform to this outdated convention; contributors should ignore that approach when introducing new tests and instead encode this information using [the `esid` frontmatter tag](#esid).
+**Note** The project is currently transitioning from a naming system based on specification section numbers. There remains a substantial number of tests that conform to this outdated convention; contributors should ignore that approach when introducing new tests and instead encode this information using [the `esid` frontmatter key](#esid).
 
 ## Test Case Style
 
 A test file has three sections: Copyright, Frontmatter, and Body.  A test looks roughly like this:
 
 ```javascript
-// Copyright (C) 2015 [Contributor Name]. All rights reserved.
+// Copyright (C) $Year $ContributorName. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
-
 /*---
- description: brief description
- info: >
-   verbose test description, multiple lines OK.
-   (info typically contains relevant, direct quote from ECMAScript)
+description: >
+    brief description, e.g. "Non-numeric input must be rejected with a TypeError"
+esid: reference to spec section, e.g. "sec-well-known-symbols"
+info: |
+    verbose test description, multiple lines OK.
+    (info should contain relevant, direct quotes from ECMAScript if possible)
+    For example:
+
+    String.fromCodePoint ( ..._codePoints_ )
+
+    5. Repeat, while _nextIndex_ &lt; _length_
+      a. Let _next_ be _codePoints_[_nextIndex_].
+      b. Let _nextCP_ be ? ToNumber(_next_).
+      d. If _nextCP_ &lt; 0 or _nextCP_ &gt; 0x10FFFF, throw a *RangeError* exception.
```

Comment:
@leobalter I pulled this text from [ecma262/spec.html](https://github.com/tc39/ecma262/blob/master/spec.html). Is the preferred style to leave formatting marks in, or should it look more like the rendered spec?
```
    String.fromCodePoint ( ...codePoints )

    5. Repeat, while nextIndex < _length_
      a. Let next be codePoints[nextIndex].
      b. Let nextCP be ? ToNumber(next).
      d. If nextCP < 0 or nextCP > 0x10FFFF, throw a RangeError exception.
```
Or perhaps it should be a hybrid, where e.g. `_` and `*` formatting marks are preserved but entities are expanded? Please advise.

Reply:
I actually prefer to read it as plain text, the info is for immediate consumption. Although some markings are easy to read, e.g. `_nextCP_` and `*RangeError*` and others might be slightly confusing: `&gt;`  (escaped chars).

To be honest, this is a preference, I celebrate if we have the info available on each test. It helps a lot reviewing and checking the purpose of each test in any future.